docs/src/content/docs/demo.mdx

```diff
@@ -151,3 +151,34 @@ import { ShowcaseQuote } from 'starlight-showcases'
     },
   ]}
 />
+
+## Profile
+
+To showcase profiles, use the [`<ShowcaseProfile>`](/components/profile/) component.
+
+import { ShowcaseProfile } from 'starlight-showcases'
+
+<ShowcaseProfile
+  entries={[
+    {
+      href: 'https://starlight.astro.build/',
+      name: 'Starlight',
+      description: 'Starlight is a documentation generator for Astro.',
+      picture:
+        'https://starlight.astro.build/_astro/hero-star.CRrHICv4_1oHYTB.webp',
+    },
+    {
+      href: 'https://github.com/HiDeoo/starlight-showcases',
+      name: '<span style="font-weight: normal">HiDeoo/</span><br>starlight-showcases',
+      description: 'Discover the Starlight Showcases components in action.',
+      picture: 'https://avatars.githubusercontent.com/u/494699?v=4',
+    },
+    {
+      href: 'https://github.com/ghost',
+      name: 'ghost',
+      description:
+        "Hi, I'm @ghost! I take the place of user accounts that have been deleted. :ghost:",
```

Comment:
3 possibilities here with different levels of complexity:
- Drop the ":ghost:" text from the description, as it's not rendered
- Replace the ":ghost:" text with the emoji
- Be able to interpret ":{image}:" in descriptions

Reply:
Or, considering users would be the one providing data and optionally fetching it, we can also let them massage the data which could be coming from GitHub and uses this syntax or somewhere else which uses a totally different syntax.